chainio/txmgr/simple.go

```diff
@@ -67,7 +67,30 @@ func (m *SimpleTxManager) WithGasLimitMultiplier(multiplier float64) *SimpleTxMa
 // If you pass in a signed transaction it will ignore the signature
 // and resign the transaction after adding the nonce and gas limit.
 // To check out the whole flow on how this works, check out the README.md in this folder
-func (m *SimpleTxManager) Send(ctx context.Context, tx *types.Transaction) (*types.Receipt, error) {
```

Comment:
can you add a comment explaining that if you send `waitForReceipt` to `false` it return a dummy tx receipt with just a tx hash? I think we would want to make it clear for someone who is using this.

Reply:
ok